config/nav.yml

```diff
@@ -72,6 +72,7 @@ nav:
             - Configuring custom domains: developer/serving/services/custom-domains.md
             - Using a custom TLS certificate for DomainMapping: developer/serving/services/byo-certificate.md
             - Configure resource requests and limits: developer/serving/services/configure-requests-limits-services.md
+            - HTTP(S) Redirect: developer/serving/services/http-option.md
```

Comment:
Why HTTP(S)? It looks like the actual doc just talks about HTTP and this is confusing.
Also, keep an eye on sentence case in titles please as mentioned in the other PR

Reply:
The redirection is from HTTP=>HTTPS

I decided to call this section `HTTPS redirection` since that's what the annotation is enabling/disabling